dlrs/clearlinux/releasenote.md

```diff
@@ -104,6 +104,10 @@ These software components were selected because they are most popular/widely use
 
 Please refer to the [Deep Learning Reference Stack guide](https://docs.01.org/clearlinux/latest/guides/stacks/dlrs.html) for detailed instructions for running the TensorFlow and Kubeflow Benchmarks on the docker images.
 
+> **Note:**
+   Although the DLRS images and dockerfiles may be modified for your needs, there are some modifications that may cause unexpected or undesirable results. For example, using the Clear Linux `swupd bundle-add` command to add packages to a Clear Linux based container may overwrite the DLRS core components. Please use care when modifying the contents of the containers.
```

Comment:
I would add a link to DLRS' core components so that users know which ones they should care about. Those components are listed in the releasenotes.

Reply:
good point!  I will add.